css/main.css

```diff
@@ -0,0 +1,293 @@
+/*hamburger script*/
```

Comment:
Can't see any @font-face usage. 

Use any font you would find for the logo and transformed text at least.